awx/main/dispatch/worker/callback.py

```diff
@@ -119,7 +164,7 @@ def _save_event_data():
                     time.sleep(delay)
                     retries += 1
                 except DatabaseError:
-                    logger.exception('Database Error Saving Job Event for Job {}'.format(job_identifier))
```

Comment:
The failed insert is now potentially multiple jobs, so we can't just pinpoint one.